### ISSUE TITLE
Stored Object: paper:openreview.qUIMnrYEPU

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed#11800]
{
  "sourceId": "openreview",
  "paperId": "qUIMnrYEPU",
  "url": "https://openreview.net/forum?id=qUIMnrYEPU",
  "title": "ConMeZO: Adaptive Directional Sampling for Gradient-Free Finetuning of Language Models",
  "authors": "Lejs Deen Behric, Liang Zhang, Bingcong Li, Kiran Koshy Thekumparampil",
  "abstract": "Zeroth\u2011order optimization (MeZO) is an attractive strategy for finetuning large language models (LLMs) because it eliminates the memory overhead of storing intermediate activations required by backpropagation. However, it converges slowly due to the inherent curse of dimensionality when searching for descent directions in the higher-dimensional parameter space of billion-scale LLMs. We propose ConMeZO, a novel zeroth\u2011order optimizer that accelerates convergence by adaptive directional sampling. Instead of drawing the direction uniformly at random, ConMeZO restricts the sampling to a cone centered around a momentum estimate. This concentrates the search in the directions where the true gradient is more likely to lie and thus reduces the effect of higher dimensions. We analytically prove that ConMeZO achieves the same worst-case convergence rate as MeZO. Empirically, when finetuning LLMs on natural language benchmarks, ConMeZO is up to 2x faster than MeZO while retaining the low\u2011memory footprint of zeroth-order methods.",
  "timestamp": "2026-04-20T18:35:12.281Z",
  "rating": "novote",
  "publishedDate": "11 Jun 2025",
  "tags": [
    "Large Language Models",
    "Finetuning",
    "Zeroth-Order",
    "Gradient Free",
    "Adaptive Directional Sampling"
  ],
  "doi": "",
  "journalName": "ES-FoMo III",
  "sourceType": "url"
}

**Comment by dmarx** [comment]
{
  "_data": {
    "sourceId": "openreview",
    "paperId": "qUIMnrYEPU",
    "url": "https://openreview.net/forum?id=qUIMnrYEPU",
    "title": "ConMeZO: Adaptive Directional Sampling for Gradient-Free Finetuning of Language Models",
    "authors": "Lejs Deen Behric, Liang Zhang, Bingcong Li, Kiran Koshy Thekumparampil",
    "abstract": "Zeroth‑order optimization (MeZO) is an attractive strategy for finetuning large language models (LLMs) because it eliminates the memory overhead of storing intermediate activations required by backpropagation. However, it converges slowly due to the inherent curse of dimensionality when searching for descent directions in the higher-dimensional parameter space of billion-scale LLMs. We propose ConMeZO, a novel zeroth‑order optimizer that accelerates convergence by adaptive directional sampling. Instead of drawing the direction uniformly at random, ConMeZO restricts the sampling to a cone centered around a momentum estimate. This concentrates the search in the directions where the true gradient is more likely to lie and thus reduces the effect of higher dimensions. We analytically prove that ConMeZO achieves the same worst-case convergence rate as MeZO. Empirically, when finetuning LLMs on natural language benchmarks, ConMeZO is up to 2x faster than MeZO while retaining the low‑memory footprint of zeroth-order methods.",
    "timestamp": "2026-04-20T18:35:12.281Z",
    "rating": "novote",
    "publishedDate": "11 Jun 2025",
    "tags": [
      "Large Language Models",
      "Finetuning",
      "Zeroth-Order",
      "Gradient Free",
      "Adaptive Directional Sampling"
    ],
    "doi": "",
    "journalName": "ES-FoMo III",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2026-04-20T18:35:12.953Z",
    "update_mode": "append",
    "issue_number": 11800
  },
  "type": "initial_state"
}
